five.js

```diff
@@ -120,6 +120,10 @@
     });
   };
 
+  five.promiseFive =function(){
```

Comment:
I like it :+1: Perhaps we could just make it `five.promise()`? Also, we're missing an opportunity to use `five()` in the resolving of this promise:( 

```javascript
return Promise.resolve(five());
```